livekit_egress.proto

```diff
@@ -117,14 +117,21 @@ enum SegmentedFileProtocol {
   HLS_PROTOCOL = 1;
 }
 
+enum SegmentedFileSuffix {
+  DEFAULT_SEGMENTED_FILE_SUFFIX = 0;
```

Comment:
let's leave INDEX as 0 and remove default. We use default in the other enums because the default might change depending on the source